docs/creating_dataset.md

```diff
@@ -270,4 +274,52 @@ In this example, `weight` is a scalar feature read from the csv, and `class` is
 **Filtering on channels**: This example also shows why filtering values by channel is useful: In this example, each subject has 2 images taken at 
  different times with different label values. By using `label_channels=["image_time_2"]`, we can use the label associated with
  the second image for all subjects.
- 
\ No newline at end of file
+
+#### Multi-label classification datasets
+Classification datasets can be multi-label, i.e. they can have more than one label associated with every sample.
+In this case, in the label column, separate the (numerical) ground truth labels with a pipe character (`|`) to 
+provide multiple ground truth labels for the sample.
```

Comment:
Should we add here somekind of warning: at this point we support multi-label classification only (i.e. one sample can belong to multiple classes) as opposed to multi-class setting? Basically just as you have written in the ReadMe